protocol/src/test/scala/io/mediachain/protocol/CborSerializationSpec.scala

```diff
@@ -217,10 +217,10 @@ object CborSerializationSpec extends BaseSpec with ScalaCheck {
         obj must haveClass[ArtefactChainCell]
       }
     }.setGen(Gen.oneOf(
-        genArtefactReferenceCell,
-        genArtefactUpdateCell,
-        genArtefactOwnershipCell,
-        genArtefactDerivationCell,
-        genArtefactCreationCell
+        genArtefactReferenceCell(),
```

Comment:
I made them into functions with default parameters, instead of just `val`s.  So that you can control what `canonical` gets referenced by the cell, etc.